sinabs/backend/dynapcnn/utils.py

```diff
@@ -409,8 +410,8 @@ def build_from_list(
         in_shape = dvs_layer.get_output_shape()
     # Find and populate dynapcnn layers
     while lyr_indx_next < len(layers):
-        if isinstance(layers[lyr_indx_next], (nn.Dropout, nn.Dropout2d, nn.Flatten)):
-            # - Ignore dropout and flatten layers
+        if isinstance(layers[lyr_indx_next], (nn.Identity, nn.Dropout, nn.Dropout2d, nn.Flatten)):
```

Comment:
We might consider dropping this if-statement completely, because now these types should already have been filtered out by the `convert_model_to_layer_list` function. However, we can also leave it in here as a safety net.

Reply:
I agree, but perhaps to ensure that this method is usable independently, it is useful to retain it. 

We could also use the `DEFAULT_IGNORED_LAYER_TYPES` here I assume? Do you foresee any potential issues in doing that?

Reply:
That's a good suggestion, thank you!

Reply:
Updated in the latest commit.